osu.Game/Graphics/UserInterface/OsuDropdown.cs

```diff
@@ -274,21 +274,40 @@ public OsuDropdownHeader()
                 CornerRadius = corner_radius;
                 Height = 40;
 
-                Foreground.Children = new Drawable[]
+                Foreground.Child = new GridContainer
                 {
-                    Text = new OsuSpriteText
+                    RelativeSizeAxes = Axes.X,
+                    AutoSizeAxes = Axes.Y,
+                    RowDimensions = new[]
```

Comment:
Was going to suggest that this isn't required, but apparently it is in this case (as `Foreground` itself is autosizing).